Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -40,6 +43,32 @@ The following is a sample attestation result:
 }
 ```
 
+**ctsProfileMatch Vs basicIntegrity**
+
+The SafetyNet Attestation API initially provided a single value called basicIntegrity to help developers determine the integrity of a device. As the API evolved, Google introduced a new, stricter check whose results appear in a value called ctsProfileMatch, which allows developers to more finely evaluate the devices on which their app is running.
+
+In broad terms, basicIntegrity gives you a signal about the general integrity of the device and its API. Rooted devices fail basicIntegrity, as do emulators, virtual devices, and devices with signs of tampering, such as API hooks.

```

Comment:
I wouldn't state that "Rooted devices fail basicIntegrity" as a fact. The main purpose of Magisk, a very popular rooting method, is to be undetected by SafetyNet.

Maybe "Many rooted devices fail basicIntegrity"?

Reply:
Sure, I'll to do this to avoid any ambiguity.